lib/engine/game/g_1849/step/buy_e_token.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1849
+      module Step
+        class BuyEToken < Engine::Step::Base
+          def actions(entity)
+            actions = super.dup
```

Comment:
What actions are you duping? You aren't inheriting from a step, just the base class.